docs/contributing/index.rst

```diff
@@ -8,33 +8,61 @@ We're so glad that you're thinking about contributing to moja global. We welcome
 
-* Get credit for your contribution
+After making your first contribution
+------------------------------------
+
+Get credit for your contribution
+================================
 
 We use the All Contributors Bot to recognize contributors.
 
-To get recognized, just add the following line to a comment after making your contribution (like submitting a pull request, replying to a question, resolving an issue, etc.):
+To get recognized, just add the following line to a comment after making your contribution (like submitting a pull request, replying to a question, resolving an issue, etc.)
+::
+    @all-contributors please add <@username> for <contributions>
 
-@all-contributors please add <@username> for <contributions>
-Replace <@username> with your GitHub username and <contributions> with any word from this list.
+Replace ``<@username>`` with your GitHub username and ``<contributions>`` with any word from this `list <https://allcontributors.org/docs/en/emoji-key>`_.
 
-After that, the @all-contributors bot will submit a PR to include you in the list of contributors. Once this PR gets accepted, you'll get added to the Contributors table in the relevant project's README.
+If you wish to know more about the ``all-contributors`` bot usage in moja global, please checkout the `Bots and Integrations section <../DeveloperWorkflow/bots_and_integrations.html>`_.
 
-Contents:
+Report Bugs, Provide Feedback or Request Features
+=================================================
 
-.. toctree::
-  :maxdepth: 1
+We welcome all kinds of bug reports, user feedback and feature requests!
 
-  code_contribution_best_practices
-  coc
-  ways_to_contribute
+If you find an easily reproducible bug and/or are experienced in reporting bugs, feel free to just open an issue on the relevant project on GitHub.
+
+We've created some issue templates to assist you in this. Please use them to create a `new issue <https://help.github.com/en/github/managing-your-work-on-github/creating-an-issue>`_ in the relevant project's repository.
+
+Pick more complex issues to work on
+===================================
+
+Now that you have your first contribution merged, you may move on to issues without ``Help for newcomers`` and ``Help Wanted`` labels. For specific areas of interest, please filter out the issues with specific labels of your interest.
+
+Review other contributions
+==========================
+
+moja global welcomes all contributors to review each others pull requests and suggest changes. You may also choose to review other contributions by following this guide `here <../DeveloperWorkflow/reviewing.html>`_.
+
+Non-code contributions to moja global
+-------------------------------------
+
+There are plenty of ways you can contribute to moja global repositories.
+Apart from contributing code, moja global would love to any kind of help for non-code related issues as well.
```

Comment:
Change this line to "moja global would love any kind of help with non-code related issues as well"